ui/src/layout/Findings/utils.jsx

```diff
@@ -1,44 +1,22 @@
 import TitleValueDisplay, { TitleValueDisplayRow } from 'components/TitleValueDisplay';
-import { formatDate, getAssetName } from 'utils/utils';
+import { formatDate } from 'utils/utils';
 
-const NAME_SORT_IDS = [
-    "asset.assetInfo.instanceID",
-    "asset.assetInfo.podName",
-    "asset.assetInfo.dirName",
-    "asset.assetInfo.imageID",
-    "asset.assetInfo.containerName"
-];
 
-export const getAssetAndScanColumnsConfigList = () => ([
+export const getScanColumnsConfigList = () => ([
     {
-        Header: "Asset name",
-        id: "assetName",
-        sortIds: NAME_SORT_IDS,
-        accessor: (data) => getAssetName(data.asset.assetInfo),
-    },
-    {
-        Header: "Asset location",
-        id: "assetLocation",
-        sortIds: ["asset.assetInfo.location"],
-        accessor: (data) => data.asset.assetInfo.location || data.asset.assetInfo.repoDigests?.[0],
-    },
-    {
-        Header: "Found on",
-        id: "foundOn",
-        sortIds: ["foundOn"],
-        accessor: original => formatDate(original.foundOn)
+        Header: "Last seen by",
+        id: "lastSeenBy",
+        sortIds: ["lastSeenBy"],
+        accessor: original => formatDate(original.lastSeenBy)
```

Comment:
I think it shouldn't be formatted as `date`, as it is the id/name of the `assetScan`, right?

Reply:
let's leave the UI changes for another PR, this PR only includes UI changes that will allow UI testing like removing fields that are no longer available